WeatherStormPower/Models/WeatherBots/RainBot.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace WeatherStormPower.Models.WeatherBots;
```

Comment:
`WeatherBots` are not just "models". Let's move this folder out the `Models` folder.

Reply:
recheck please other classes in the models folders, some also don't belong here. Like the monitor.